index.html

```diff
@@ -12,32 +12,20 @@
 <body>
   <div class="container">
     <div class="row">
-      <div class="col-lg-6 col-md-12">
-        <!-- ko foreach: toDoLists -->
-        <div class="row toDoList">
-          <!-- ko component: 'toDoList' -->
-          <!-- /ko -->
-        </div>
-        <button class="btn btn-block btn-danger" data-bind="click: $parent.removeToDoList">Remove</button>
+      <!-- ko foreach: widgets -->
+      <div class="col-lg-3 col-md-4 col-sm-6 col-xs-12">
+        <!-- ko component: name -->
         <!-- /ko -->
+        <button class="btn btn-block btn-danger remove-widget" data-bind="click: $parent.remove">Remove</button>
+      </div>
+      <!-- /ko -->
+    </div>
+    <div class="row">

```

Comment:
Here we have a second bootstrap row. This contains the buttons to add new types of widgets. I decided I want the column size to be the same as for widgets because it looks good like this.